x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -171,3 +174,22 @@ export function argsForProcess(passedEvent: ResolverEvent): string | undefined {
   }
   return passedEvent?.process?.args;
 }
+
+/**
+ * used to sort events
+ */
+export function orderByTime(first: ResolverEvent, second: ResolverEvent): number {
```

Comment:
@jonathan-buttner this now does a tie breaker with an arbitrary comparison of event ID.

Reply:
Nice! Thanks!